run_dir/design/flowcells.html

```diff
@@ -201,20 +215,38 @@ <h1><span id="page_title">Flowcells</span>
       var title = $('#fc_table thead th').eq( $(this).index() ).text();
       $(this).html( '<input size=10 type="text" placeholder="Search '+title+'" />' );
     } );
+    $('#ont_fc_table tfoot th').each( function () {
+      var title = $('#ont_fc_table thead th').eq( $(this).index() ).text();
+      $(this).html( '<input size=10 type="text" placeholder="Search '+title+'" />' );
+    } );
 
     var table = $('#fc_table').DataTable({
       "paging":false,
       "info":false,
       "order": [[ 0, "desc" ]]
     });
+    var table = $('#ont_fc_table').DataTable({
+      "paging":false,
+      "info":false,
+      "order": [[ 0, "desc" ]]
+    });
```

Comment:
This will break the Illumina table. Consider initialising the ONT table in a different function or better yet, put the table init code in `ont_flowcells.html`, so everything is in one place.

Reply:
e556687b8d5f17ab0522a9a82587003c751adbb0